### PR TITLE
Add new tests

### PR DESCRIPTION
**Pull request by MrBanananator**

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 1]
```diff
@@ -121,6 +123,61 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsidered2() {
```
Comment (Marine5174-zz) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java]: Prefer to name as an explicit test case. The naming pattern is 
testCase_behavior_expectedResult 

The format might be flexible, but the name should be descriptive.

For example, this test mighe be:
public void query_multipleAttendees_allScheduled() {}

Might be worthwhile putting some thinking to name the test case, especically to distinguish from other test cases.

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 2]
```diff
@@ -121,6 +123,61 @@ public void everyAttendeeIsConsidered() {
...
+    
+    request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
+        Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, TIME_0800AM, false),
+            TimeRange.fromStartEnd(TIME_0830AM, TIME_0900AM, false),
+            TimeRange.fromStartEnd(TIME_0930AM, TimeRange.END_OF_DAY, true));
+
+    Assert.assertEquals(expected, actual);
```
Comment (Marine5174-zz) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:151]: I think Assert has an assertThat you might be able to use, such as:

assertThat(actual).containsExactElementsIn(...).inOrder();

The exact signature might be slightly different. But worth reading a bit into the assert framework.

Comment (MrBanananator) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:151]: I'm not too sure what you are saying here. The current assert already works right?

Comment (Marine5174-zz) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java:151]: Yes but we usually prefer literal expected results, as it is clearer to the readers.

**Review by Marine5174-zz — APPROVED** [review]

**Pull request merged**